cli/parse_pdfs.py

```diff
@@ -90,20 +90,28 @@ def parse_file(
     elif ocr_agent == "gcv":
         ocr_agent = lp.GCVAgent()
 
+    num_pages = len(pdf_images)
+
     all_pages_metadata = []
     all_text_blocks = []
 
     for page_idx, image in tqdm(
-        enumerate(pdf_images), total=len(pdf_images), desc=pdf_path.name
+        enumerate(pdf_images), total=num_pages, desc=pdf_path.name
     ):
         # If running in visual debug mode and the pdf is large, randomly select pages to save images for to avoid excessive redundancy
         # and processing time
         if debug:
-            if len(pdf_images) > 10:
```

Comment:
Should this sit in a function to make the code more readable and clear? 

def select_pages_subset():
    ...